templates/default/indicators.html

```diff
@@ -2,29 +2,29 @@
 {if ConfigHelper::CheckPrivilege('helpdesk_administration') || ConfigHelper::CheckPrivilege('helpdesk_operation')}
 <a href="?m=indicators&action=critical" class="lms-ui-indicator" title="{trans("Critical tickets")}" id="indicator-critical" style="display: none;">
     <span class="lms-ui-indicator-counter">0</span>
-    <img src="img/24-ring-red.png">
+    <i class="lms-ui-indicator-icon lms-ui-icon-critical" style="size: 1.5em"></i>
```

Comment:
lepiej dodać jakąś klasę powiększającą tak jak ma już FA np. mają fa-2x, fa-3x to nam brakuje fa-1-5x ;-)

Reply:
Zrobiłem fa-2x